packages/web/src/components/basic/ReactiveComponent.d.ts

```diff
@@ -7,7 +7,7 @@ export interface ReactiveComponentProps {
 	render?: (...args: any[]) => any;
 	componentId: string;
 	aggregationField?: string;
-	size: number;
+	size?: number;
```

Comment:
@ShahAnuj2610 You have to use this `size` prop in the ReactiveComponent, if not using then remove it.

Reply:
it's being used -> https://github.com/appbaseio/reactivecore/blob/3d4d01163941ec89261982ab2e180adffc4862c5/src/utils/helper.js#L454